File: FlightServiceAPI/Models/Flight.cs
using System.ComponentModel.DataAnnotations;

namespace FlightServiceAPI.Models
{
    public partial class Flight
    {
        [Key]
        public int Id { get; set; }
        public string FlightNumber { get; set; }
        public string Destination { get; set; }
        public DateTime DepartureDateTime { get; set; }
        public DateTime ArrivalDateTime { get; set; }
        public string DepartureAirport { get; set; }
        public string ArrivalAirport { get; set; }
        public int MaxCapacity { get; set; }

        public virtual ICollection<Passenger> Passengers { get; set; }
    }
}

File: FlightServiceClient/flight-service-client/src/app/flight/flight.ts
import { Passenger } from "../passenger/passenger";

export interface Flight {
    id: number;
    flightNumber: string;
    destination: string;
    departureDateTime: Date;
    arrivalDateTime: Date;
    departureAirport: string;
    arrivalAirport: string;
    maxCapacity: number;
    passengers: Passenger[];
}

export interface FlightDTO {
    flightNumber: string;
    destination: string;
    departureDateTime: Date;
    arrivalDateTime: Date;
    departureAirport: string;
    arrivalAirport: string;
    maxCapacity: number
}